Engine/src/Application.bf

```diff
@@ -11,6 +11,7 @@ namespace SteelEngine
 
 		private Window _window ~ delete _;
 		private Window.EventCallback _eventCallback = new => OnEvent ~ delete _;
+		private GLFWInputSystem _inputSystem = new GLFWInputSystem() ~ delete _;
```

Comment:
With the introduction of Systems in #20, can we rename this to `GLFWInputManager` or something of that sort?